posts/investigating-spirv-for-the-shader-system.md

```diff
@@ -1,12 +1,13 @@
 ---
-title: "Investigating spir-v for the shader system"
+title: "Investigating spir-v for the shader system - Part 1"
 author: youness
 popular: false
 image: /images/spir.png
 tags: ['.NET', 'spir-v', 'shader', 'hlsl', 'glsl', 'sdsl', 'compilation']
```

Comment:
Let's keep just major tags, as these are for a better navigation in this page https://www.stride3d.net/tags/. I would keep: .NET, Shaders.

We can add more tags later, if there will be more blog posts specific/related to some topic.

Shaders maybe, probably as plural, as we have there: Particles, Prefabs,

Reply:
Yes of course!
That makes sense, i'll make the changes!